apps/ledger-live-desktop/src/storyly/useStoryly.tsx

```diff
@@ -39,22 +39,21 @@ export const useStoryly = (
       token: storyly.params?.stories[instanceId].token || "",
       lang: language,
       segments: [`lang_${language}`],
-      events: {
-        isReady: data => {
-          dataRef.current = data;
-          // Triggered when story is ready.
-        },
-        actionClicked: story => {
-          if (story?.media?.actionUrl) {
-            openURL(story.media.actionUrl);
-            ref.current?.close?.();
-            dispatch(closeAllModal());
-            setDrawer();
-          }
-        },
-      },
       props: { ...props, ...options?.styleProps },
     });
+
+    ref.current?.on("loaded", (data: StorylyData) => {
+      dataRef.current = data.groupList;
+    });
+
+    ref.current?.on("actionClicked", (story: Story) => {
+      if ("actionUrl" in story) {
+        openURL(story.actionUrl as string);
```

Comment:
Doesn't typescript infer that the actionUrl is a string here ? It would be better to avoid using `as`

Reply:
obviously typescript didn't because of the `"actionUrl" in story` check . i will fix it